src/Perpetuum/EntityFramework/EntityFactory.cs

```diff
@@ -66,6 +66,43 @@ public Entity Create(EntityDefault entityDefault,IIDGenerator<long> idGenerator)
 
             return entity;
         }
-    }
-}
+        public Entity CreateForData(string definitionName, IIDGenerator<long> idGenerator) {
+            bool forData = true;
```

Comment:
`fordata` check seems unnecessary as this whole function is for that purpose right?

Reply:
Correct. I can research to see if manually unpacking the robot and initializing it will have the same results - but I was worried that any additional "touching" of the entity would make the game server realize it's not actually attached to anyone and throw errors. I'll research.

Reply:
Removed this and replaced with Robot.Unpack.